src/csv2rdf/source.clj

```diff
@@ -58,7 +63,11 @@
   (get-json [f] (read-json f))
 
   String
-  (get-json [s] (json/read-str s)))
+  (get-json [s] (json/read-str s))
+
+  Path
+  (get-json [p]
+    (get-json (slurp (io/reader (Files/newInputStream p (into-array OpenOption [StandardOpenOption/READ])))))))
```

Comment:
Would `(read-json (Files/newInputStream p (into-array OpenOption [StandardOpenOption/READ])))` also work here?

Reply:
Oh thanks I meant to check that, but forgot; and yes it works.

Reply:
Fixed and forced pushed